spec/rubocop/cop/layout/space_inside_reference_brackets_spec.rb

```diff
@@ -109,6 +109,20 @@
       RUBY
     end
 
+    it 'registers an offense when ref bct w/lead spc assigned by ref bct' do
```

Comment:
Just let the description trail two lines instead of scrambling the sentence. 🙂

Reply:
Trust me, it pained me to abbreviate these words. 😀 Was just following the style in the rest of the file. I'll fix this.